taxi_service/settings.py

```diff
@@ -44,6 +44,7 @@
     "django.contrib.staticfiles",
     "debug_toolbar",
     "crispy_forms",
+    "crispy_bootstrap4",
```

Comment:
The `CRISPY_TEMPLATE_PACK` setting is defined twice. Please remove the duplicate to avoid confusion and ensure the settings are correctly applied.